addon/helpers/filter-by.js

```diff
@@ -0,0 +1,57 @@
+import Ember from 'ember';
+
+const {
+  Helper,
+  computed: { filter, filterBy },
+  defineProperty,
+  get,
+  isArray,
+  isPresent,
+  observer,
+  set
+} = Ember;
+
+export default Helper.extend({
+  compute([array, byPath, value]) {
+    set(this, 'array', array);
+    set(this, 'byPath', byPath);
+    set(this, 'value', value);
+
+    return get(this, 'content');
+  },
+
+  array:   null,
```

Comment:
Small nitpick: prefer not to align like this, makes adding/removing keys a bit annoying